src/seedu/addressbook/Main.java

```diff
@@ -13,117 +13,131 @@
-            ui.showInitFailedMessage();
-            /*
-             * ==============NOTE TO STUDENTS=========================================================================
-             * We are throwing a RuntimeException which is an 'unchecked' exception. Unchecked exceptions do not need
-             * to be declared in the method signature.
-             * The reason we are using an unchecked exception here is because the caller cannot reasonably be expected
-             * to recover from an exception.
-             * Cf https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html
-             * =======================================================================================================
-             */
-            throw new RuntimeException(e);
-        }
-    }
-
-    /** Prints the Goodbye message and exits. */
-    private void exit() {
-        ui.showGoodbyeMessage();
-        System.exit(0);
-    }
-
-    /** Reads the user command and executes it, until the user issues the exit command.  */
-    private void runCommandLoopUntilExitCommand() {
-        Command command;
-        do {
-            String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
-
-        } while (!ExitCommand.isExit(command));
-    }
-
-    /** Updates the {@link #lastShownList} if the result contains a list of Persons. */
-    private void recordResult(CommandResult result) {
-        final Optional<List<? extends ReadOnlyPerson>> personList = result.getRelevantPersons();
-        if (personList.isPresent()) {
-            lastShownList = personList.get();
-        }
-    }
-
-    /**
-     * Executes the command and returns the result.
-     * 
-     * @param command user command
-     * @return result of the command
-     */
-    private CommandResult executeCommand(Command command)  {
-        try {
-            command.setData(addressBook, lastShownList);
-            CommandResult result = command.execute();
-            storage.save(addressBook);
-            return result;
-        } catch (Exception e) {
-            ui.showToUser(e.getMessage());
-            throw new RuntimeException(e);
-        }
-    }
-
-    /**
-     * Creates the StorageFile object based on the user specified path (if any) or the default storage path.
-     * @param launchArgs arguments supplied by the user at program launch
-     * @throws InvalidStorageFilePathException if the target file path is incorrect.
-     */
-    private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorageFilePathException {
-        boolean isStorageFileSpecifiedByUser = launchArgs.length > 0;
-        return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
-    }
-
+	/** Version info of the program. */
```

Comment:
this commit seems unrelated to the current activity?
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.